src/api/userActivity.ts

```diff
@@ -0,0 +1,58 @@
+import axios from 'axios';
+import queryString from 'query-string';
+
+import { ENDPOINTS } from './constants';
+
+interface UserActivity {
+  userId: number | string;
+}
+
+interface PoadmapList extends UserActivity{
+  filter:string; 
+  page?:number;
+  pageSize?:number
+}
+
+export const userActivityList = async ({
+  userId,
+}: UserActivity) => {
+  const { data } = await axios.get(
+    `${ENDPOINTS.USERS}${userId}${ENDPOINTS.STATISTICS}`
+  );
+
+  return data;
+};
+
+export const userRoadmapList = async ({
+  userId,
+  filter,
+  page, pageSize = 6
+}: PoadmapList) => {
+  const query = queryString.stringify(
+    {
+      status: filter,
+      page_size: pageSize,
+      page: page,
+    },
+    {
+      skipEmptyString: true,
+    },
+  );
+
+  const { data } = await axios.get(
+    `${ENDPOINTS.USERS}${userId}${ENDPOINTS.ROADMAPS}?${query}`
+  );
+
+  return data;
+};
+
+export const deleteRoadmap = async ({
+  userId,
+  roadmapId
```

Comment:
Usually function name with verb because it to do something. Like in 49 line. So I suggest change `userActivityList` and `userRoadmapList` to more clear name like `getSomething`